packages/apps/docs/src/components/MostPopular/MostPopular.tsx

```diff
@@ -0,0 +1,26 @@
+import { BrowseSection } from '../BrowseSection';
+
+import { useFetchMostPopularPages } from './useFetchMostPopularPages';
+
+import Link from 'next/link';
+import React from 'react';
+
+const MostPopular = (): JSX.Element => {
+  const { data } = useFetchMostPopularPages();
```

Comment:
we could do this call in getStaticProps
that way it is done in build time instead of seperate api call.